AdyenDropIn/Utilities/ComponentManager+PaymentComponentBuilder.swift

```diff
@@ -316,11 +316,14 @@ private extension ComponentManager {
     }
 
     func createBancontactComponent(with paymentMethod: BCMCPaymentMethod) -> PaymentComponent? {
-        // TODO: these will be also be replaced by the factories, no need to update
-        BCMCComponent(
+        var cardConfiguration = configuration.card.cardComponentConfiguration
```

Comment:
This was required to fix the failing `testBCMCPaymentMethod()` test that check if localization parameters were passed. Of course, it will replaced later.

Reply:
I've brought it back